plugin/trino-memsql/src/main/java/io/trino/plugin/memsql/MemSqlClient.java

```diff
@@ -292,6 +299,37 @@ public boolean isLimitGuaranteed(ConnectorSession session)
         return true;
     }
 
+    @Override
+    public Optional<PreparedQuery> implementJoin(
+            ConnectorSession session,
+            JoinType joinType,
+            PreparedQuery leftSource,
+            PreparedQuery rightSource,
+            List<JdbcJoinCondition> joinConditions,
+            Map<JdbcColumnHandle, String> rightAssignments,
+            Map<JdbcColumnHandle, String> leftAssignments)
+    {
+        if (joinType == JoinType.FULL_OUTER) {
```

Comment:
I didn't want to static import these, because there can be multiple different ways of referring to joins and Client is not join-specific.
All am saying is that it was conscious, but still am open to suggestions